components/availability/src/index.html

```diff
@@ -52,6 +52,7 @@
 
         component.calendars = calendars;
 
+        // component.email_ids = ["pooja.g@nylas.com", "phil.r@nylas.com"];
```

Comment:
✂️

Reply:
I'm going to be updating this when I add tests. This is intended for now